lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgstr "Blackout intr\xe1ffat. Aterst\xe1lla utskr?"
+
+# MSG_CALIBRATING_HOME c=20
+#: ultralcd.cpp:8492
+msgid "Calibrating home"
+msgstr "Kalibrerar hem"
+
+# MSG_CALIBRATE_BED c=18
+#: ultralcd.cpp:5787
+msgid "Calibrate XYZ"
+msgstr "Kalibrerar XYZ"
+
+# MSG_HOMEYZ c=18
+#: messages.c:52
+msgid "Calibrate Z"
+msgstr "Kalibrera Z"
+
+# MSG_CALIBRATE_PINDA c=17
+#: ultralcd.cpp:4529
+msgid "Calibrate"
+msgstr "Kalibrerar"
+
+# MSG_CANCEL2 c=10
+#: messages.c:20
+msgid ">Cancel"
+msgstr ">Avbryt"
+
+# MSG_MOVE_CARRIAGE_TO_THE_TOP c=20 r=8
+#: ultralcd.cpp:3318
+msgid "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
+msgstr "Kalibrerar XYZ. Vrid vredet f\xefr att flytta Z-vagnarna upp till \xe1ndstoppen. Klicka n\xe1r du \xe1r klar."
+
+# MSG_CALIBRATE_Z_AUTO c=20 r=2
+#: messages.c:21
+msgid "Calibrating Z"
+msgstr "Kalibrerar Z"
+
+# MSG_HOMEYZ_DONE c=20
+#: ultralcd.cpp:656
+msgid "Calibration done"
+msgstr "Kalibraring utf\xefrd"
+
+# MSG_MENU_CALIBRATION c=18
+#: messages.c:68
+msgid "Calibration"
+msgstr "Kalibrering"
+
+# MSG_SD_REMOVED c=20
+#: ultralcd.cpp:8939
+msgid "Card removed"
+msgstr "Kort borttaget"
+
+# MSG_CHECKING_FILE c=17
+#: ultralcd.cpp:8580
+msgid "Checking file"
+msgstr "Kontrollerar fil"
+
+# MSG_NOT_COLOR c=19
+#: ultralcd.cpp:2565
+msgid "Color not correct"
+msgstr "F\xe1rg ej korrekt"
+
+# MSG_COOLDOWN c=18
+#: messages.c:27
+msgid "Cooldown"
+msgstr "Kyla ner"
+
+# MSG_COPY_SEL_LANG c=20 r=3
+#: ultralcd.cpp:4435
+msgid "Copy selected language?"
+msgstr "Kopiera det valda spraket?"
+
+# MSG_CRASHDETECT c=13
+#: messages.c:30
+msgid "Crash det."
+msgstr "Krockdetekt."
+
+# MSG_CHOOSE_FIL_1ST_LAYERCAL c=20 r=7
+#: ultralcd.cpp:4842
+msgid "Choose a filament for the First Layer Calibration and select it in the on-screen menu."
+msgstr "V\xe1lj ett filament f\xefr f\xefrsta lagrets kalibrering och v\xe1lj det i sk\xe1rmmenyn."
+
+# MSG_CRASH_DETECTED c=20
+#: messages.c:29
+msgid "Crash detected."
+msgstr "Krock uppt\xe1ckt."
+
+# MSG_CRASH_RESUME c=20 r=3
+#: Marlin_main.cpp:651
+msgid "Crash detected. Resume print?"
+msgstr "Krock uppt\xe1ckt. Ateruppta utskrift?"
+
+# MSG_CRASH c=7
+#: messages.c:28
+msgid "Crash"
+msgstr "Krock"
+
+# MSG_CURRENT c=19
+#: ultralcd.cpp:5929
+msgid "Current"
+msgstr "Aktuell"
+
+# MSG_DATE c=17
+#: ultralcd.cpp:1985
+msgid "Date:"
+msgstr "Datum:"
+
+# MSG_COMMUNITY_MADE c=18
+#: messages.c:25
+msgid "Community made"
+msgstr "Allm\xe1nhetsgjord"
+
+# MSG_DISABLE_STEPPERS c=18
+#: ultralcd.cpp:5678
+msgid "Disable steppers"
+msgstr "Inaktivera stepper"
+
+# MSG_BABYSTEP_Z_NOT_SET c=20 r=12
+#: messages.c:13
+msgid "Distance between tip of the nozzle and the bed surface has not been set yet. Please follow the manual, chapter First steps, section First layer calibration."
+msgstr "Avstandet mellan munstyckets spets och b\xe1ddytan har inte st\xe1llts in \xe1nnu. V\xe1nligen f\xeflj manualen F\xefrsta lagrets kalibrering."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.